pkg/cluster/cluster.go

```diff
@@ -133,6 +134,10 @@ func new(config Config, e *spec.EtcdCluster, stopC <-chan struct{}, wg *sync.Wai
 		if err := c.createClientServiceLB(); err != nil {
 			return nil, fmt.Errorf("fail to create client service LB: %v", err)
 		}
+	} else {
+		if !c.bm.isBackupSetup() {
+			return nil, errors.New("")
```

Comment:
An obvious problem is in https://github.com/coreos/etcd-operator/issues/534#issuecomment-269369175, where we have race to have an empty cluster trying to poll backup service again and again, but in the end it keeps failing with "DNS" problem. But in fact we never created backup RS and service!

Instead, we make it less mystic and more obvious to check it at the beginning.